include/staff/templates/users.tmpl.php

```diff
@@ -34,9 +34,9 @@
 $page=($_GET['p'] && is_numeric($_GET['p']))?$_GET['p']:1;
 $pageNav=new Pagenate($total,$page,PAGE_LIMIT);
 $qstr = '&amp;'. Http::build_query($qs);
-$qs += array('sort' => $_REQUEST['sort'], 'order' => $_REQUEST['order']);
+$qs += array('id' => $org->getId(), 'sort' => $_REQUEST['sort'], 'order' => $_REQUEST['order']);
 
-$pageNav->setURL('users.php', $qs);
+$pageNav->setURL(basename($_SERVER['PHP_SELF']), $qs);
```

Comment:
Where is a changed line above that that you see here ;)

Please look at the whole change in "Files Changed" section before commenting.